web/src/components/home/subcomponents/song_of_the_week/messages.js

```diff
@@ -0,0 +1,7 @@
+const messages = {
+  title: "Song of the Week",
+  listenOnYoutube: "YouTube",
+  listenOnSpotify: "Spotify",
+};
```

Comment:
why do you separate these in to a separate file if they're so small and not re-used anywhere? That seems a bit overengineered

Reply:
You seem a bit overengineered

Reply:
Consistency really. Separation of concerns (template vs text vs styling)